hack/generate/csv.sh

```diff
@@ -60,14 +60,22 @@ image "networking-istio" "${registry}/knative-v$(metadata.get dependencies.net_i
 image "istio-webhook__webhook" "${registry}/knative-v$(metadata.get dependencies.net_istio):net-istio-webhook"
 
 image "eventing-controller__eventing-controller"    "${eventing}-controller"
+image "eventing-controller__eventing-controller__kube-rbac-proxy"    "$rbac_proxy"
 image "sugar-controller__controller"                "${eventing}-sugar-controller"
+image "sugar-controller__controller__kube-rbac-proxy"               "$rbac_proxy"
 image "eventing-webhook__eventing-webhook"          "${eventing}-webhook"
+image "eventing-webhook__eventing-webhook__kube-rbac-proxy"          "$rbac_proxy"
 image "storage-version-migration-eventing-eventing-$(metadata.get dependencies.eventing)__migrate" "${eventing}-storage-version-migration"
 image "mt-broker-controller__mt-broker-controller"  "${eventing}-mtchannel-broker"
+image "mt-broker-controller__mt-broker-controller__kube-rbac-proxy"  "$rbac_proxy"
 image "mt-broker-filter__filter"                    "${eventing}-mtbroker-filter"
+image "mt-broker-filter__filter__kube-rbac-proxy"                    "$rbac_proxy"
 image "mt-broker-ingress__ingress"                  "${eventing}-mtbroker-ingress"
+image "mt-broker-ingress__ingress__kube-rbac-proxy"                  "$rbac_proxy"
 image "imc-controller__controller"                  "${eventing}-channel-controller"
+image "imc-controller__controller__kube-rbac-proxy"                 "$rbac_proxy"
 image "imc-dispatcher__dispatcher"                  "${eventing}-channel-dispatcher"
+image "imc-dispatcher__dispatcher__kube-rbac-proxy"                  "$rbac_proxy"
```

Comment:
`__XXX` already indicates a container name, so I think all of these have to change to 

 ```suggestion
image "imc-dispatche__kube-rbac-proxy"                  "$rbac_proxy"
```

(i.e. remove the first container name)

Reply:
can we do that "legacy" clean up in a separate PR ? 

I am happy to clean up all those "dups" - ok ?

Reply:
ah, nevermind ...   Ignore my comment ... that's the rbac_proxy ...

Reply:
but... they are lke: https://github.com/openshift-knative/serverless-operator/pull/964/files#diff-da13200d4ae862c053922b38b3bf7dfdc1bd6f52518c2f160156fe41c119bf39R62-R79

Hrm....

Reply:
Yes, these are wrong too, that's how we noticed :joy:

Reply:
:rofl:  ok...